addons/refuel/CfgVehicles.hpp

```diff
@@ -451,31 +437,26 @@ class CfgVehicles {
         };
     };
     class Land_FlexibleTank_01_F: FlexibleTank_base_F {
-        transportFuel = 0; //300
         GVAR(hooks)[] = {{0, 0, 0.5}};
         GVAR(fuelCargo) = 300;
     };
 
     // Vanilla buildings
     class Land_Fuelstation_Feed_F: House_Small_F {
-        transportFuel = 0; //50k
         GVAR(hooks)[] = {{0,0,-0.5}};
         GVAR(fuelCargo) = REFUEL_INFINITE_FUEL;
     };
 
     class Land_fs_feed_F: House_Small_F {
-        transportFuel = 0; //50k
         GVAR(hooks)[] = {{-0.4,0.022,-.23}};
         GVAR(fuelCargo) = REFUEL_INFINITE_FUEL;
     };
 
     class Land_FuelStation_01_pump_F: House_F {
-        transportFuel = 0; //50k
         GVAR(hooks)[] = {{0, 0.4, -0.5}, {0, -0.4, -0.5}};
         GVAR(fuelCargo) = REFUEL_INFINITE_FUEL;
     };
     class Land_FuelStation_01_pump_malevil_F: House_F {
-        transportFuel = 0; //50k
```

Comment:
`setFuelCargo` won't work on map objects
e.g. the fuel station on stratis (`[2713, 5796]`) will fill a empty car

Reply:
I knew this command is somehow broken.

Reply:
Will check that. Hiding and replacing it with a simple object might be an option.

Reply:
> `setFuelCargo` won't work on map objects
> e.g. the fuel station on stratis (`[2713, 5796]`) will fill a empty car

It does work, you must just handle all 3 pumps at the station.
The real problem here is that fuel pumps are `Static` children and they don't trigger `InitPost` EH. This can be handled with `nearestObjects` or `nearObjects` call with each pump class at mission init. But how to handle pumps created during mission with `createVehicle` after mission init?
At least fuel pumps `class CBA_Extended_EventHandlers` removed in #4533 can be restored. But this must be done also for all 3rd party pumps so some compat can be needed.

Reply:
terrain objects can also stream in/out, and might not store their fuel state inbetween.

Reply:
I already tested this here https://github.com/acemod/ACE3/pull/7613#issuecomment-612692751